workshops/migrations/0119_auto_20160929_1131.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-09-29 16:31
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workshops', '0118_auto_20160922_0009'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='membership',
+            name='variant',
+            field=models.CharField(choices=[('partner', 'Partner'), ('affiliate', 'Affiliate'), ('sponsor', 'Sponsor'), ('bronze', 'Bronze'), ('silver', 'Silver'), ('gold', 'Gold'), ('platinum', 'Platinum')], max_length=40),
```

Comment:
This is independent of the `Sponsorship` model, isn't it?

Memberships represent contracts between an org and SWC/LC.
Sponsorships represent contracts between an org and a particular event.